### PR TITLE
Added tag support to table creation

### PR DESCRIPTION
**Pull request by matllubos**
Added support of AWS tags

### CODE REVIEW

File: pynamodb/models.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
     BATCH_WRITE_PAGE_LIMIT,
     META_CLASS_NAME, REGION, HOST, NULL,
     COUNT, ITEM_COUNT, KEY, UNPROCESSED_ITEMS, STREAM_VIEW_TYPE,
-    STREAM_SPECIFICATION, STREAM_ENABLED, BILLING_MODE, PAY_PER_REQUEST_BILLING_MODE
+    STREAM_SPECIFICATION, STREAM_ENABLED, BILLING_MODE, PAY_PER_REQUEST_BILLING_MODE, Tags
```
Comment (ikonst) [inline pynamodb/models.py:36]: shouldn't this be capitals?

File: pynamodb/connection/table.py [hunk 2]
```diff
@@ -304,6 +304,7 @@ def create_table(
         local_secondary_indexes: Optional[Any] = None,
         stream_specification: Optional[Dict] = None,
         billing_mode: str = DEFAULT_BILLING_MODE,
+        tags: Optional[Dict] = None,
```
Comment (ikonst) [inline pynamodb/connection/table.py:307]: Per https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/TaggingRestrictions.html:
```suggestion
        tags: Optional[Dict[str, str]] = None,
```

File: pynamodb/models.py (hunk 1 above)
Comment (matllubos) [reply pynamodb/models.py]: Fixed

File: pynamodb/connection/table.py (hunk 2 above)
Comment (matllubos) [reply pynamodb/connection/table.py]: Fixed

**Comment by matllubos** [comment]
I added the test

**Review by garrettheel — APPROVED** [review]

**Comment by garrettheel** [comment]
Thanks @matllubos!

**Pull request merged**